editions/tw5.com/tiddlers/widgets/TextWidget.tid

```diff
@@ -9,6 +9,9 @@ text-test: e=mc^^2^^
 
 The text widget displays plain text without parsing it as [[WikiText]], opposite of [[WikifyWidget]].
 
+* The ~TextWidget can be used to suppress a soft link from being recognised see [[Hard and Soft Links]]
```

Comment:
Recommend that the words "soft link" be the link to the tiddler "Hard and Soft Links", dropping the text "See [[Hard and Soft Links]]".